.github/workflows/conformance-datapath.yaml

```diff
@@ -139,13 +139,15 @@ jobs:
             tunnel: 'vxlan'
             encryption: 'wireguard'
             lb-mode: 'snat'
+            endpoint-routes: 'true'
```

Comment:
Any chance this could be enabled with one of the IPsec cases? Endpoint routes matter for IPsec unfortunately.

Reply:
Ideally both with tunneling and native routing, but if that's too much, I would just cover native routing.

Reply:
5.10 and 5.15?

Reply:
Are you proposing to enable IPsec on those two? Sounds good to me but it means WireGuard won't be tested with endpoint routes. Maybe that's fine?

Reply:
Nobody stops us from having a few VM instances with the same kernel vsn but different configurations :-)

Reply:
Oh, sure. That works too. I just thought you wanted to keep the number of VM limited :slightly_smiling_face:

Reply:
Could you open an issue for that? :pray:

Reply:
https://github.com/cilium/cilium/issues/23090